cimr/processor/utils.py

```diff
@@ -420,12 +434,13 @@ class Integrator:
     Parameters:
     -----------
 
+    data_type: {'gwas', 'eqtl', 'sqtl', 'pqtl', 'twas', 'tad'}
     file_name: name of the file containing the data
-    data_type = {'gwas', 'eqtl', 'tad'}
-    can_be_public: boolean variable indicating whether the contributed data
-                   can be released as a part of the public archive.
-                   for cimr >= 0.2.3, can_be_public parameter will determine
-                   the destination repository of the contributed data
+    data_type = default.DATA_TYPES
```

Comment:
Isn't `DATA_TYPES` imported from `defaults.py`?

Reply:
great catch! thank you!